planning/planning_validator/autoware_planning_validator/src/node.cpp

```diff
@@ -279,7 +279,7 @@ void PlanningValidatorNode::displayStatus()
   warn(s->is_valid_latency, "planning component latency is larger than threshold!!");
   warn(s->is_valid_yaw_deviation, "planning trajectory yaw difference from ego yaw is too large!!");
   warn(s->is_valid_trajectory_shift, "planning trajectory had sudden shift!!");
-  warn(s->is_valid_collision_check, "planning trajectory leads to collision!!");
+  warn(s->is_valid_intersection_collision_check, "planning trajectory leads to collision!!");
```

Comment:
As various collision detection features may be added in the future, it would be helpful if error messages clearly indicate the specific scene or context in which the collision was detected.

```suggestion
  warn(s->is_valid_intersection_collision_check, "planning trajectory leads to collision!! (objects within intersection)");
```